src/main/resources/application.conf

```
@@ -21,3 +21,6 @@ db.hikari."dataSource.password"="root"
 # connection pool configuration
 db.hikari.maximumPoolSize=9
 db.hikari.leakDetectionThreshold=30000
+
+# base url for images
+img_url = "https://image.tmdb.org/t/p/original/"
```

Je mettrais ça dans un champ api, ça permet de rendre ta conf évolutive et de regrouper l'ensemble de la configuration api ensemble.

```suggestion
api = {
    tmdb = {
        image.base-url =  "https://image.tmdb.org/t/p/original/"
    }
}
```

Ainsi si jamais tu voulais ajouter une autre api tu pourrais, mieux encore si tu voulais ajouter une autre configuration pour les images tu ferais ainsi :


```
api = {
    tmdb = {
        image = {
           base-url =  "https://image.tmdb.org/t/p/original/"
           autre-conf = "mon autre conf"
        } 
    }
}
```